webapp/loggedin/surveyViews/taxonomyEdit/components/taxaTableRow.js

```diff
@@ -1,22 +1,20 @@
 import React from 'react'
 
-import Taxon from '../../../../../core/survey/taxon'
-
 const TaxaTableRow = props => {
 
   const { row, idx, offset, vernacularLanguageCodes } = props
 
   return (
     <>
       <div>{idx + offset + 1}</div>
-      <div>{Taxon.getCode(row)}</div>
-      <div>{Taxon.getFamily(row)}</div>
-      <div>{Taxon.getGenus(row)}</div>
-      <div>{Taxon.getScientificName(row)}</div>
+      <div>{row.code}</div>
+      <div>{row.family}</div>
+      <div>{row.genus}</div>
+      <div>{row.scientific_name}</div>
```

Comment:
why?

Reply:
I'll restore the code from the previous version